### ISSUE TITLE
Upload to the KDE Store

### ISSUE THREAD

**Issue by Jak2k** [Donnnno/Arcticons-Linux#10]
Please upload the pack to the [KDE Store](https://store.kde.org/browse/). This allows users to easily discover, download and install the pack.

**Comment by autinerd** [comment]
I will look into it, thanks for the suggestion!

**Comment by Donnnno** [comment]
Good idea, but maybe we need to have a version with all the basics first, before we reach a bigger audience.
